packages/cheqd-sdk/src/ledger/CheqdSdkLedgerService.ts

```diff
@@ -0,0 +1,107 @@
+import type { AbstractCheqdSDKModule, CheqdSDK, ICheqdSDKOptions, DidStdFee, DIDDocument } from '@cheqd/sdk'
+import type { SignInfo } from '@cheqd/ts-proto/cheqd/did/v2'
+import type { MsgCreateResourcePayload } from '@cheqd/ts-proto/cheqd/resource/v2'
+
+import { injectable } from '@aries-framework/core'
+import { createCheqdSDK, DIDModule, ResourceModule, CheqdNetwork } from '@cheqd/sdk'
+import { DirectSecp256k1HdWallet, DirectSecp256k1Wallet } from '@cosmjs/proto-signing'
+import { EnglishMnemonic as _ } from '@cosmjs/crypto'
+import { fromString } from 'uint8arrays'
```

Comment:
We have a `TypedArrayEncoder`  that generally handles transformign from/to UInt8arrays and buffers. Can we leverage that you think?

Reply:
There are differences in all these outputs. The first output is not compatible with our sdk

TypedArrayEncoder.toBase64(key.publicKey)
ZZ8zBCapsnUeEXMfD+xpDNeMQQ49asVY069I466pmnQ= 

TypedArrayEncoder.toBase64URL(key.publicKey)
ZZ8zBCapsnUeEXMfD-xpDNeMQQ49asVY069I466pmnQ

toString(key.publicKey, 'base64')
ZZ8zBCapsnUeEXMfD+xpDNeMQQ49asVY069I466pmnQ

Reply:
I think the `TypedArrayEncoder` supports both Buffer and Uint8Arrays, and this shouldn't happen. There's probably a bug in either of the implementation then? Seems most of it is the same. I'll look into the `toString` implementation. 

Do you know if `uint8array` works in react native?

Reply:
Sorry It's a bug in our sdk. Fixed it